ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java

```diff
@@ -62,4 +68,30 @@ private ConnectExecutor() { }
     return response.error()
         .map(err -> new ErrorEntity(statement.getStatementText(), err));
   }
+
+  private static void throwIfConnectorIsDuplicate(final CreateConnector createConnector,
+      final ConnectClient client) {
+    if (createConnector.isNotExists()) {
```

Comment:
bug: doesn't this need to be reversed?   This returns true if the user included `IF NOT EXIST` in the statement.

```suggestion
    if (!createConnector.isNotExists()) {
```

Reply:
It would be right to proceed with checking if the supplied connector exists only if  `IF NOT EXISTS` is in the query right ? else we could just avoid the check , meaning
```
if `IF NOT EXISTS`
   checkDuplicate
else
   continue
```

Reply:
If the `IF NOT EXISTS` is set, then it should not cause an error if the connector already exists. Btw, I found out the `CREATE STREAM|TABLE IF NOT EXISTS` does not work as expected either (https://github.com/confluentinc/ksql/issues/6050).

Reply:
@hemantgs, #6067 has been merged!

Reply:
Fixed this , as per suggestions
a query with IF NOT EXISTS will return 

```
Message                               
---------------------------------------
 Connector CALLS_READER already exists 
---------------------------------------
```
if connector exists